### PR TITLE
New package: WinsiderSS.SystemInformer version 3.0.11299.6772

### LINKED ISSUE #109907: [Package Request]: WinsiderSS.SystemInformer
### How can we help?

None

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://systeminformer.sourceforge.io/nightly.php
Publisher: Winsider Seminars & Solutions, Inc.
Package Name: System Informer
Description: A free, powerful, multi-purpose tool that helps you monitor system resources, debug software and detect malware.
Package Version: 3.0.6772
Installer URL: https://github.com/winsiderss/si-builds/releases/download/3.0.6772/systeminformer-3.0.6772-setup.exe

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- Resolve #109907

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111530&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
I have no idea to resolve this

**Comment by stephengillie** [comment]
> I have no idea to resolve this

Unfortunately, this might not be possible to fix. It would have to be reported as not malware.

File: manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: WinsiderSS.SystemInformer
+PackageVersion: 3.0.11299.6772
+Installers:
+- Architecture: x86
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.installer.yaml:8]: Install is not silent. Please add the silent install switches

File: manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: WinsiderSS.SystemInformer
+PackageVersion: 3.0.11299.6772
```
Comment (Trenly) [inline manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.installer.yaml:5]: Package writes `3.0` to the registry, and the GitHub tag is `3.0.6772`. This should either be -
1) Updated to 3.0 -or-
2) Changed to 3.0.6772 and the `AppsAndFeaturesEntries` added with `DisplayVersion`

File: manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: WinsiderSS.SystemInformer
+PackageVersion: 3.0.11299.6772
+PackageLocale: en-US
+Publisher: Seminars & Solutions, Inc
```
Comment (Trenly) [inline manifests/w/WinsiderSS/SystemInformer/3.0.11299.6772/WinsiderSS.SystemInformer.locale.en-US.yaml:7]:
```suggestion
Publisher: System Informer
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
Please retry one time.
If it still appears this, this PR should be blocked temporarily

**Comment by CoolPlayLin** [comment]
Don't assign it to me, I have no idea to fix

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @CoolPlayLin,

Is there an installer switch to bypass the message box above, and complete the install automatically?

**Comment by CoolPlayLin** [comment]
@stephengillie
Sorry, I haven't used this software yet. I don't how to bypass it.

CC @0xThiebaut

**Comment by CoolPlayLin** [comment]
？

**Comment by 0xThiebaut** [comment]
Currently OoO, will investigate/request the switch on Sept 26.

**Comment by CoolPlayLin** [comment]
@stephengillie Add `Blocking-issue` temporarily

**Comment by 0xThiebaut** [comment]
Blocked by https://github.com/winsiderss/systeminformer/issues/759 .

**Comment by CoolPlayLin** [comment]
@stephengillie Could this be not marked as [No-Recent-Activity](https://github.com/microsoft/winget-pkgs/labels/No-Recent-Activity) ?

**Comment by CoolPlayLin** [comment]
Why it added again?

**Comment by CoolPlayLin** [comment]
Why it added again?

**Comment by CoolPlayLin** [comment]
No adding :)

**Comment by CoolPlayLin** [comment]
Stop

**Comment by CoolPlayLin** [comment]
.

**Comment by CoolPlayLin** [comment]
Stop

**Pull request closed without merging**